src/main/java/seedu/address/logic/commands/AddMeetingCommand.java

```diff
@@ -30,17 +31,18 @@ public class AddMeetingCommand extends Command {
             + PREFIX_DATE + "DATE "
             + "[" + PREFIX_START_TIME + "START_TIME]"
             + "[" + PREFIX_END_TIME + "END_TIME]\n"
-            + "[" + PREFIX_NAME + "NAME]...\n"
+            + "[" + PREFIX_NAME + "NAME]..."
+            + "[" + PREFIX_GROUP + "GROUP]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_MEETING_NAME + "CS2103T Meeting "
             + PREFIX_DATE + "2020-10-10 "
             + PREFIX_START_TIME + "1000 "
             + PREFIX_END_TIME + "1200 "
             + PREFIX_NAME + "Alice "
-            + PREFIX_NAME + "Bob";
+            + PREFIX_NAME + "Bob"
```

Comment:
Might need a whitespace after "Bob" to make sure the next string is separated properly